labgrid/driver/networkusbstoragedriver.py

```diff
@@ -49,6 +51,52 @@ def on_activate(self):
     def on_deactivate(self):
         pass
 
+    @step(args=['filenames', 'target_dir', 'partition'])
+    def copy_files(self, filenames, target_dir, partition=1):
+        if not self.storage.path:
+            raise ExecutionError(
+                "{} is not available".format(self.storage_path)
+            )
+
+        dev_path = pathlib.PurePath(self.storage.path + str(partition))
+        mount_path = pathlib.PurePath('/media') / dev_path.name
```

Comment:
Is this correct everywhere, or is this configurable / different by distribution? Or is there a straightforward way to get the actual path at runtime?

Reply:
[The mount path is determined by pmount](https://linux.die.net/man/1/pmount) and cannot be configured; therefore it should be consistent over different distributions. I do not see any method to get the path at runtime.

Reply:
You can actually set the media dir in [configure](https://bazaar.launchpad.net/~fourmond/pmount/main/view/head:/configure.ac#L27), but I guess assuming `/media/` is okay.

Reply:
Ok, I was not aware of that. I added a constant + comment for this path so that we are at least aware of that it can be changed theoretically.